src/pal/src/eventprovider/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 2.8.12.2)
+
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+project(eventprovider)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)   
+  
+  add_definitions(-DPAL_STDCPP_COMPAT=1)
+  include_directories(${COREPAL_SOURCE_DIR}/inc/rt)
+  add_compile_options(-fPIC)
+
+
+   add_library(eventprovider1
+"lttngeventprovMicrosoft_Linux_DotNETRuntime.cpp"
```

Comment:
Yes we need them to be separate for different logging systems, presumably each OS can have multiple logging systems. I will move these files under a folder named after a logging system